pymc/sampling.py

```diff
@@ -2125,18 +2125,18 @@ def draw(
 
     Parameters
     ----------
-    vars
+    vars : Variable or iterable of Variable
         A variable or a list of variables for which to draw samples.
-    draws : int
-        Number of samples needed to draw. Detaults to 500.
-    mode
-        The mode used by ``aesara.function`` to compile the graph.
-    **kwargs
-        Keyword arguments for :func:`pymc.aesara.compile_pymc`
+    draws : int, default 1
+        Number of samples needed to draw.
+    mode : str or aesara.compile.mode.Mode, optional
+        The mode used by :func:`aesara.function` to compile the graph.
```

Comment:
is this fine now?

Reply:
I think so but we need to wait for the readthedocs preview to be generated and check that links are correctly generated.

Reply:
These two links are working, the Variable and compile_pymc ones need extra work on a different pr to work, they are fine like this for now

Reply:
Thank You : )